docs/enviroment-variables.adoc

```diff
@@ -0,0 +1,60 @@
+:page-title: @ClearEnvironmentVariable and @SetEnvironmentVariable
+:page-description: JUnit Jupiter extensions to clear and set the values of environment variable
+
+The `@ClearEnvironmentVariable` and `@SetEnvironmentVariable` annotations can be used to clear, respectively, set the values of environment variables for a test execution.
+Both annotations work on the test method and class level, are repeatable as well as combinable.
+After the annotated method has been executed, the variables mentioned in the annotation will be restored to their original value or will be cleared if they didn't have one before.
+Other environment variables that are changed during the test, are *not* restored.
+
+Warning: Your SecurityManager has to allow modifications.
```

Comment:
I think we need a clearer warning that this uses reflection and may technically break on any minor Java release or on different OSes. Users should be aware that while this may be the best way to test env variables, it's not actually a good one. ;)

Reply:
You mentioned  in #174 
> - put a warning into the .adoc
> - put a warning into the Javadoc
> - print a warning once per test suite/class/method (?) execution if the extension is active

I did not know excatly how much you wanted here (and where for the last point).
Would at be ok if you do that?

Reply:
Sure.